### PR TITLE
Lesson 3: Mailroom.py

### PR DESCRIPTION
**Pull request by stevetwalker**
Uploading the final assignment for lesson 3!

### CODE REVIEW

File: students/steve_walker/lesson03/mailroom.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    print(" "+"-"*60+"\n Donor Name"+" "*10+
+        "| Total Given | Num Gifts | Average Gift\n "+"-"*60)
+    
+    # Generate and print donor summaries
+    donor_summary = []
+    for record in donor_records:
+        donor_summary.append([record[0], sum(record[1],0), 
+            len(record[1]), sum(record[1],0)/len(record[1])])
+    for i in donor_summary:
+        print(" {0:<20} ${1:>11.2f}{2:>12}  ${3:>12.2f}".format(*i))
```
Comment (charlesrobison) [inline students/steve_walker/lesson03/mailroom.py]: Per assignment donors should be sorted by total donated, descending.

**Comment by stevetwalker** [comment]
Ah, I'm so sorry. I totally missed that requirement. Do I have a chance to
resubmit?

**Comment by natasha-aleksandrova** [comment]
yes of course.

Create a new PR with the sorting change, let us know you're resubmitting for full grade via comment and we will adjust your grade.

**Comment by stevetwalker** [comment]
Wonderful, thank you! I've submitted the updated. No rush--have a lovely
next few days!

**Pull request merged**